src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -44,7 +44,7 @@
 #include "ps_kernel.h"
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`